sources/router.h

```diff
@@ -51,14 +51,20 @@ od_router_reconfigure(od_router_t *, od_rules_t *);
 int
 od_router_expire(od_router_t *, od_list_t *);
 void
-od_router_gc(od_router_t *);
+od_router_gc(od_router_t *router);
```

Comment:
Well, we did not have names in prototypes. I don't think it is right. But let's keep things alligned - either all with names, or all without.